### ISSUE TITLE
www.mvs4u.co

### ISSUE THREAD

**Issue by doomlerd** [PreMiD/Presences#865]
**Prerequisites and essential questions** 
* [x] Is it a popular site?
* [x] Is the website older than 2 months? 
* [ ] Is the site locked to a specific country/region?
* [ ] Is the site a paid service? (e.g. Netflix, Hulu)
* [ ] Does the website feature NSFW content? (e.g. porn, etc...)
* [ ] Are you a donator/patron?
* [x] Do you acknowledge that coding presences is completely voluntary and may take time for your service to be added regardless of priority?

**What's your Discord username?** 
`Re - 8sh#9002`
**What's the name of the service?** 
movs4u.tv / mvs4u.co
**What should the Presence display?** 
Current playing movies/tv episodes and status of the other pages
**If possible, please provide a logo for the service (512x512 minimum)** 
basically 9anime's logo but with orange color.
